### ISSUE TITLE
Trashed documents show up on site

### ISSUE THREAD

**Issue by ericanastas** [nytimes/library#357]
### Context (Environment)

Node 18.14.0
Folder source

### Expected Behavior

If documents are moved to the trash, and no longer show up in a folder on the google drive interface, I would expect the document to no longer show up on the site as well.

### Actual Behavior

Trashed documents continue to show up on the site until they are permanently deleted.

![image](https://user-images.githubusercontent.com/1441899/224193875-09350cb6-212c-4904-9127-b87621ca93f2.png)

![image](https://user-images.githubusercontent.com/1441899/224193974-ab272f29-9103-4f50-ab84-9aecc704f75c.png)

### To Reproduce

Move a document to the trash. 

### Possible Solution

This [`getOptions()` in search.js](https://github.com/nytimes/library/blob/bb0010ec57b5711ed42ac3997f304b14dcc756c3/server/search.js#L90) includes `AND trashed = false`, however this filter is not included in[ the corresponding `getOptions()` function in list.js](https://github.com/nytimes/library/blob/bb0010ec57b5711ed42ac3997f304b14dcc756c3/server/list.js#L105).

These options should include the `trashed = false` filter as well.

**Comment by afischer** [comment]
Hi @ericanastas, thanks for the report. 

As a workaround until this is fixed, you can create a top-level folder called `Trash` and move files you wish to hide on the site there. This was originally implemented as user permissions around deleting files can be messy within shared drives.

**Comment by ericanastas** [comment]
Can you elaborate on how the permissions for deleting files can be "messy" in GSDs? Maybe there's something I'm not considering. My understanding is only drive managers can permanently delete files from the trash right? I think most users would expect files that have been moved to the trash to no longer show up on the site.

**Comment by afischer** [comment]
I'm not sure if this is still the case, but it used to be that users that did not originally own files in shared drives could not delete them, even if they were able to edit or move them. 

It's very possible Google has changed the way that permissions work for files in a shared drive in the past few years, so It's definitely worth revisiting.

**Comment by JacksonRoberts-Git** [comment]
Hi! Is there any chance I could claim or otherwise be assigned to this bug? I am relatively new to GitHub but I want to see if I can help you with this

**Comment by JacksonRoberts-Git** [comment]
Me again. I am working on a project for class (software engineering) and I need a GitHub task to complete. Not sure if this makes you more or less likely to assign me to this, but I hope you understand where I'm coming from. Thanks.

**Comment by ericanastas** [comment]
> I'm not sure if this is still the case, but it used to be that users that did not originally own files in shared drives could not delete them, even if they were able to edit or move them.
> 
> It's very possible Google has changed the way that permissions work for files in a shared drive in the past few years, so It's definitely worth revisiting.

@afischer 

This is true. Only managers can permanently delete files, but regular users can still move files to this trash. 

https://support.google.com/a/users/answer/12380484?hl=en

![image](https://github.com/user-attachments/assets/f9abc641-f14d-4716-bbe8-b0432aa4f730)
